### PR TITLE
Bigtable: Create Instanceadmin sample

### PR DESCRIPTION
**Pull request by sangramql**
Add instanceadmin sample for python

### CODE REVIEW

**Comment by sduskis** [comment]
@billyjacobson, can you PTAL?

File: bigtable/instanceadmin/instanceadmin.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+Prerequisites:
+- Create a Cloud Bigtable project.
+  https://cloud.google.com/bigtable/docs/
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+
+Operations performed:
+- Create a Cloud Bigtable Instance.
+- List Instance for a Cloud Bigtable.
+- Delete a Bigtable Instance.
```
Comment (billyjacobson) [inline bigtable/instanceadmin/instanceadmin.py]: It should be Cloud Bigtable for all of these.

File: bigtable/instanceadmin/instanceadmin.py [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+from google.cloud import bigtable
+from google.cloud.bigtable import enums
+
+
+def run_instance_operations(project_id, instance_id):
+    ''' Check Instance exists.
+        Creates a Production instance with default Cluster.
+        List instances in a project.
+        List cluster in an instance.
```
Comment (billyjacobson) [inline bigtable/instanceadmin/instanceadmin.py]: s/cluster/clusters

File: bigtable/instanceadmin/instanceadmin.py [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    else:
+        print 'Instance {} already exists.'.format(instance_id)
+    # [END bigtable_check_instance_exists]
+
+    # [START bigtable_create_prod_instance]
+    cluster = instance.cluster("ssd-cluster1", location_id=location_id,
+                               serve_nodes=serve_nodes,
+                               default_storage_type=storage_type)
+    if not instance.exists():
+        print '\nCreating a Instance'
```
Comment (billyjacobson) [inline bigtable/instanceadmin/instanceadmin.py]: s/Creating a/Creating an

File: bigtable/instanceadmin/instanceadmin.py [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    instance = client.instance(instance_id)
+    cluster = instance.cluster(cluster_id)
+
+    # [START bigtable_delete_cluster]
+    print '\nDeleting Cluster'
+    if cluster.exists():
+        cluster.delete()
+        print 'Cluster deleted: {}'.format(cluster_id)
+    else:
+        print '\nCluster {} does not exists.'.format(cluster_id)
```
Comment (billyjacobson) [inline bigtable/instanceadmin/instanceadmin.py]: s/exists/exist

File: bigtable/instanceadmin/README.rst [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+To run this sample:
+
+.. code-block:: bash
+
+    $ python instanceadmin.py
+
+    usage: instanceadmin.py [-h] [run] [dev-instance] [del-instance] [add-cluster] [del-cluster] project_id instance_id cluster_id
+
+    Demonstrates how to connect to Cloud Bigtable and get .
```
Comment (billyjacobson) [inline bigtable/instanceadmin/README.rst]: This line cuts off mid sentence. Not really sure what happened. Maybe try regenerating it?

File: bigtable/instanceadmin/instanceadmin.py (hunk 1 above)
Comment (sangramql) [reply bigtable/instanceadmin/instanceadmin.py]: Fixed.

File: bigtable/instanceadmin/instanceadmin.py (hunk 2 above)
Comment (sangramql) [reply bigtable/instanceadmin/instanceadmin.py]: Fixed.

File: bigtable/instanceadmin/instanceadmin.py (hunk 4 above)
Comment (sangramql) [reply bigtable/instanceadmin/instanceadmin.py]: Fixed.

File: bigtable/instanceadmin/README.rst (hunk 5 above)
Comment (sangramql) [reply bigtable/instanceadmin/README.rst]: Fixed.

**Pull request merged**